src/include/common/object_pool.h

```diff
@@ -4,7 +4,6 @@
 #include <string>
 #include <utility>
 #include "common/allocator.h"
-#include "common/container/concurrent_queue.h"
```

Comment:
This was an unnecessary include that propagated to quite a few .cpp files in the system.